tests/test_sqs/test_sqs.py

```diff
@@ -171,10 +195,18 @@ def test_create_fifo_queue_with_high_throughput():
 
 
 @mock_aws
-def test_create_queue():
+@pytest.mark.parametrize(
+    "q_name",
+    [
+        str(uuid4())[0:6],
```

Comment:
The SQS tests are run in parallel, and each parallel process collects all test-names. After execution, the results of all processes are merged and checked for consistency. 

I think the error that we're seeing is that the test-names are not consistent across all test processes, due to this randomization. If I understand it correctly, this could be fixed by explicitly specifying the test-id's:
```
ids=["random", "underscores", "hyphens", "combined"]
```
As long as the test-ids are equal across every run, it should not be a problem.

Reply:
If you have `pytest-xdist` installed, you can run `pytest tests/test_sqs -n 4` locally to validate this theory